src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -421,6 +433,24 @@ private void processEnd(
         doEnd();
     }
 
+    private void processAbort(
+            DirectBuffer buffer,
+            int index,
+            int length)
+    {
+        factory.correlations.remove(acceptCorrelationId);
+        if (targetBeginIssued)
+        {
+            factory.writer.doAbort(target, targetId);
+            cleanupConnectReply.run();
+        }
+        else
+        {
+            factory.writer.doAbort(acceptReply, replyStreamId);
```

Comment:
This `ABORT` of the accept reply stream needs to happen unconditionally, right?
For example, what if you get an `ABORT` while the connection is idle between requests?